platform/kube/inject/inject.go

```diff
@@ -132,12 +132,19 @@ const (
 )
 
 // InitImageName returns the fully qualified image name for the istio
-// init image given a docker hub and tag
-func InitImageName(hub, tag string) string { return hub + "/proxy_init:" + tag }
+// init image given a docker hub and tag and debug flag
```

Comment:
Why add a debug flag here? Seem it is not used

Reply:
because eventually we will have debug/opt for all images we produce